### PR TITLE
Gdr 1955

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Updated logic for excess
Related JIRA issue: GDR-1955

## Why was it changed?
To unify colnames

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: R/generate_wrappers.R [hunk 1]
```diff
@@ -425,10 +371,9 @@ prepareCodilutionData <- function(df, df_layout) {
   colnames(df) <- paste0(colnames(df), "_2")
   df_2 <- cbind(df_layout, df)
   df_2 <- df_2[df_2$DrugName != df_2$DrugName_2, ]
-  rows <- df_2$Concentration_2 > 0
+  rows <- which(df_2$Concentration_2 > 0)
```
Comment (gladkia) [inline R/generate_wrappers.R:374]: safer solution?

Comment (bczech) [reply R/generate_wrappers.R:374]: `DT` requires indexes for rows instead of boolean values

Comment (gladkia) [reply R/generate_wrappers.R:374]: Should we add the task to evaluate if there are other such cases left in our code?

Comment (bczech) [reply R/generate_wrappers.R:374]: Yes, we can evaluate it

Comment (gladkia) [reply R/generate_wrappers.R:374]: Please add the task.

File: R/generate_wrappers.R [hunk 2]
```diff
@@ -8,22 +8,18 @@ generateNoNoiseRawData <- function(cell_lines, drugs, save = TRUE) {
...
-    if (save) {
-      save_rds(
-        rdsObj = mae,
-        rdsName = "finalMAE_small_no_noise.RDS"
-      )
-    }
-    invisible(mae)
-  } else {
-    df_merged
   }
```
Comment (j-smola) [inline R/generate_wrappers.R:21]:
```
} else {
    df_merged
  }
```
Was it intentionally deleted?

File: R/generate_wrappers.R [hunk 3]
```diff
@@ -75,24 +66,19 @@ generateLigandData <- function(cell_lines, drugs, save = TRUE) {
...
-    if (save) {
-      save_rds(
-        rdsObj = mae,
-        rdsName = "finalMAE_wLigand.RDS"
-      )
-    }
-    invisible(mae)
-  } else {
-    df_merged
   }
```
Comment (j-smola) [inline R/generate_wrappers.R:80]: As above.

File: R/generate_wrappers.R [hunk 4]
```diff
@@ -35,23 +31,18 @@ generateNoiseRawData <- function(cell_lines, drugs, save = TRUE) {
...
-    if (save) {
-      save_rds(
-        rdsObj = mae,
-        rdsName = "finalMAE_small.RDS"
-      )
-    }
-    invisible(mae)
-  } else {
-    df_merged
   }
```
Comment (j-smola) [inline R/generate_wrappers.R:44]: As above.

File: R/generate_wrappers.R (hunk 2 above)
Comment (bczech) [reply R/generate_wrappers.R:21]: Yes, because I removed ` if (requireNamespace("gDRcore", quietly = TRUE))` that is always true in this package

Comment (j-smola) [reply R/generate_wrappers.R:21]: Thx  😄

**Review by j-smola — APPROVED** [review]

**Pull request merged**